ci-operator/step-registry/cucushift/installer/rehearse/azure/aro/provision/cucushift-installer-rehearse-azure-aro-provision-ref.metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+	"path": "cucushift/installer/rehearse/azure/aro/provision/cucushift-installer-rehearse-azure-aro-provision-ref.yaml",
```

Comment:
cucushift/installer/rehearse folder is for saving cluster install workflow yaml files, probably we need to place the step in ci-operator/step-registry folder (e.g: ci-operator/step-registry/azure/provision/aro).

Reply:
I've moved the aro vnet and cluster provision code to step-registry/aro/provision